core/src/wsv.rs

```diff
@@ -295,7 +295,8 @@ impl<W: WorldTrait> WorldStateView<W> {
     ///
     /// # Errors
     /// - No such [`Asset`]
-    /// - No [`Account`] to which the [`Asset`] is associated.
+    /// - No [`Account`] with which the [`Asset`] is associated.
+    /// - No [`Domain`] with which the [`Account`] is associated.
```

Comment:
```suggestion
    /// - The [`Asset`] is not associated with any [`Account`].
    /// - The [`Account`] is not associated with any [`Domain`].
```

Reply:
It's not that it's not associated with any account/domain, it's that specific account/domain it's supposed to be associated with doesn't exist.

Would "Assotiated X doesn't exist" be better?

Reply:
```suggestion
    /// - The [`Account`] with which the [`Asset`] is associated doesn't exist. 
    /// - The [`Domain`] with which the [`Account`] is associated doesn't exist.
```